### PR TITLE
Update tabcordion

### PR DESCRIPTION
**Pull request by jaortiz**

### CODE REVIEW

File: components/tabcordion/examples/50-local-tokens.js [hunk 1]
```diff
@@ -0,0 +1,116 @@
+/** @jsx jsx */
+
+import { forwardRef } from 'react';
+import { GEL, jsx } from '@westpac/core';
+import { Tab, Tabcordion } from '@westpac/tabcordion';
+import { data } from './_data';
+
+const Panel = forwardRef(({ isLast, isSelected, ...props }, ref) => {
+	const common = {
```
Comment (dominikwilkowski) [inline components/tabcordion/examples/50-local-tokens.js]: Not to be silly but can't we just put the thing in there? What's the point of the `common` var here?

File: components/tabcordion/examples/50-local-tokens.js [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+		backgroundColor: 'black',
+		color: 'white',
+		padding: 16,
+	};
+	return <div ref={ref} css={common} {...props} />;
+});
+
+const TabItem = ({ isSelected, isJustified, isLast, ...props }) => {
+	const common = {
```
Comment (dominikwilkowski) [inline components/tabcordion/examples/50-local-tokens.js]: Same as above... This is harder for me to connect

File: components/tabcordion/examples/50-local-tokens.js [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+		fontSize: '1rem',
+		justifyContent: 'space-between',
+		outline: 0,
+		padding: '0.75rem 1.125rem',
+		position: 'relative',
+		textAlign: 'left',
+		width: '100%',
+	};
+
+	const lastStyles =
```
Comment (dominikwilkowski) [inline components/tabcordion/examples/50-local-tokens.js]: Same ☝️

File: components/tabcordion/examples/50-local-tokens.js [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+		Panel,
+		TabItem,
+		AccordionLabel,
+	};
+
+	return (
+		<GEL brand={overridesWithTokens}>
+			<h3>Responsive</h3>
+			<Tabcordion>
+				{data.map(t => (
```
Comment (dominikwilkowski) [inline components/tabcordion/examples/50-local-tokens.js]: I'm actively removing automated layouting from example files so it's easier to read. Let's repeat things so people can the the actual accordion API

File: components/tabcordion/src/Tabcordion.js [hunk 5]
```diff
@@ -1,9 +1,10 @@
 import React, { Children, useEffect, useRef, useState } from 'react';
 import PropTypes from 'prop-types';
+import { useBrand, merge } from '@westpac/core';
 import { useContainerQuery } from '@westpac/hooks';
-
-import { TabItem, TabRow } from './styled';
+import { TabItem, TabRow, AccordionLabel, Panel } from './styled';
```
Comment (dominikwilkowski) [inline components/tabcordion/src/Tabcordion.js]: We need to talk about `styled.js`.

File: components/tabcordion/src/Tabcordion.js [hunk 6]
```diff
@@ -1,9 +1,10 @@
...
 import { Tab } from './Tab';
+import pkg from '../package.json';
 
 let instanceId = 0;
```
Comment (dominikwilkowski) [inline components/tabcordion/src/Tabcordion.js]: We need to make sure this works when the component is run multiple times on the same page even in different react wrappers (multiple react apps)

File: components/tabcordion/src/Tabcordion.js [hunk 7]
```diff
@@ -87,13 +98,13 @@ export const Tabcordion = props => {
 	const tabCount = Children.count(props.children);
 
 	return (
-		<div ref={containerRef}>
+		<div ref={containerRef} css={{ ...overrides.CSS }}>
```
Comment (dominikwilkowski) [inline components/tabcordion/src/Tabcordion.js]: Nice but we said we would lowercase `css` when it stands alone: https://github.com/WestpacGEL/GEL/tree/develop#local-tokens-naming-convention-overwrites

File: components/tabcordion/src/styled.js [hunk 8]
```diff
@@ -17,78 +17,62 @@ export const TabRow = forwardRef((props, ref) => (
...
+				textAlign: 'left',
+				textDecoration: 'none',
+				transition: 'background .3s ease',
+				width: '100%',
+				cursor: 'pointer',
+
+				'&:last-child': {
+					marginRight: 0,
+				},
+				...styles[appearance],
```
Comment (dominikwilkowski) [inline components/tabcordion/src/styled.js]: `appearance` should be `look` now: https://github.com/WestpacGEL/GEL/tree/develop#props-api-vocabulary

File: components/tabcordion/examples/50-local-tokens.js (hunk 1 above)
Comment (jaortiz) [reply components/tabcordion/examples/50-local-tokens.js]: Yea sorry I forgot to move it into the element itself. I find it easier to move out the styles when I'm working on a component and then move them back when im done.

File: components/tabcordion/src/Tabcordion.js (hunk 5 above)
Comment (jaortiz) [reply components/tabcordion/src/Tabcordion.js]: Are we moving all styled components into the component file that requires them?

File: components/tabcordion/src/Tabcordion.js (hunk 6 above)
Comment (jaortiz) [reply components/tabcordion/src/Tabcordion.js]: How do we want to handle unique id's for components? You said before you didnt like generating a random unique id (was using shortid package in other components to do this.)

File: components/tabcordion/src/Tabcordion.js (hunk 5 above)
Comment (dominikwilkowski) [reply components/tabcordion/src/Tabcordion.js]: We should chat about this. I personally like this more :)

File: components/tabcordion/src/Tabcordion.js (hunk 6 above)
Comment (dominikwilkowski) [reply components/tabcordion/src/Tabcordion.js]: We need to add the unique ID to the `window` object directly.

**Comment by jaortiz** [comment]
I've fixed most of the PR comments. Only ones I haven't done are the ones that require further discussion:
- styled.js
- unique id in the window object

File: components/tabcordion/src/Tab.js [hunk 9]
```diff
@@ -1,50 +1,61 @@
-import React, { forwardRef } from 'react';
+/** @jsx jsx */
+
+import React, { useState, useEffect, forwardRef } from 'react';
```
Comment (dominikwilkowski) [inline components/tabcordion/src/Tab.js]: `useEffect` is declared but never used

File: components/tabcordion/src/Tab.js [hunk 10]
```diff
@@ -1,50 +1,61 @@
...
+import { jsx, useBrand, merge, wrapHandlers } from '@westpac/core';
```
Comment (dominikwilkowski) [inline components/tabcordion/src/Tab.js]: `wrapHandlers` is declared but never used

**Pull request merged**